pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pipeline-ai"
-version = "2.0.0"
+version = "2.0.2"
```

Comment:
Note have skipped [v2.0.1](https://github.com/mystic-ai/pipeline/tree/v2.0.1) as that tag already exists.